test_hessian/src/main/java/test/TestCustomDecode.java

```diff
@@ -166,6 +167,11 @@ public Object customArgTypedFixedList_Object() throws Exception {
         return Arrays.deepEquals(list, (Object[]) o);
     }
 
+    public Object customArgTypedFixedList_Integer() throws Exception {
```

Comment:
the method name should remove `TypedFixedList` ,i think you can change other method name customArgTypedFixedList_Decimal together

Reply:
both `customArgTypedFixedList_Decimal ` and `customArgTypedFixedList_DateNull` fixed together